src/Umbraco.Core/Composing/IAsyncComponent.cs

```diff
@@ -0,0 +1,38 @@
+namespace Umbraco.Cms.Core.Composing;
+
+/// <summary>
+/// Represents a component.
+/// </summary>
+/// <remarks>
+/// <para>
+/// Components are created by DI and therefore must have a public constructor.
+/// </para>
+/// <para>
+/// All components are terminated in reverse order when Umbraco terminates, and disposable components are disposed.
+/// </para>
+/// <para>
+/// The Dispose method may be invoked more than once, and components should ensure they support this.
+/// </para>
+/// </remarks>
```

Comment:
If the manual disposing is removed, we can get rid of these remarks (as MS DI is the default for resolving services).

Reply:
```suggestion
/// <remarks>
/// <para>
/// Components are created by DI and therefore must have a public constructor.
/// </para>
/// <para>
/// All components are terminated in reverse order when Umbraco terminates, and disposable components are disposed.
/// </para>
/// </remarks>
```